dev-java/bcprov/bcprov-1.65.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+JAVA_PKG_IUSE="doc source test"
+
+inherit java-pkg-2 java-pkg-simple
+
+MY_P="${PN}-jdk15on-${PV/./}"
+
+DESCRIPTION="Java cryptography APIs"
+HOMEPAGE="http://www.bouncycastle.org/java.html"
+SRC_URI="http://www.bouncycastle.org/download/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="1.54"
```

Comment:
You haven't changed the SLOT. You'll see that all the older versions have their own SLOT. It's hard to remember now but the API probably broke a lot. @monsieurp did try to put it back to SLOT 0 but then ran into some difficulty and went back on that idea. He probably remembers the details better.

Maybe this version is compatible with 1.54 but you'd have to try all the matching revdeps to find out whether this would immediately break anything... luckily that's just net-p2p/freenet. Is that the one you already tried?

Regardless of what happens with the SLOT now, you're not going to be able to clean up the old versions in a hurry because a whole bunch of other revdeps will need non-trivial fixing. :disappointed:

Reply:
Yeah, this was unintentional. I might chat to @monsieurp just to see if he remembers what the breakage was.

It looks like upstream Freenet is on 1.59. So nobody seems to keep up with this. 

Bleh, I had a feeling it wouldn't be simple!